docker-app/Dockerfile

```diff
@@ -12,14 +12,20 @@ ARG PIP_DISABLE_PIP_VERSION_CHECK=1
 ARG PIP_NO_CACHE_DIR=1
 
 # install psycopg2 requirements
-RUN apt-get update && apt-get install -y libpq-dev python3-dev gcc
+RUN apt update \
+    && apt install -y \
+    libpq-dev \
+    python3-dev \
+    gcc \
+    && rm -rf /var/lib/apt/lists/*
 
 # install `pip-compile` (as part of `pip-tools`)
 RUN pip install pip-tools
```

Comment:
Shall we make this one also pip3, I think it's the only place where we use pip over pip3?